numpy/core/code_generators/generate_umath.py

```diff
@@ -891,24 +892,25 @@ def make_arrays(funcdict):
             thedict = chartotype1  # one input and one output
 
         for t in uf.type_descriptions:
-            if t.func_data not in (None, FullTypeDescr) and not isinstance(t.func_data, FuncNameSuffix):
+            if (t.func_data not in (None, FullTypeDescr) and
+                    not isinstance(t.func_data, FuncNameSuffix)):
                 funclist.append('NULL')
                 astype = ''
                 if not t.astype is None:
                     astype = '_As_%s' % thedict[t.astype]
-                astr = '%s_functions[%d] = PyUFunc_%s%s;' % \
-                       (name, k, thedict[t.type], astype)
+                astr = ('%s_functions[%d] = PyUFunc_%s%s;' %
+                           (name, k, thedict[t.type], astype))
```

Comment:
Probably the rule in this case should be `least surprise`, and you have been surprised ;)